maps/map_files/BigRed/BigRed.dmm

```diff
@@ -32882,7 +32914,26 @@
 	},
 /area/bigredv2/caves/mining)
 "ohg" = (
-/obj/vehicle/train/cargo/trolley,
+/obj/structure{
```

Comment:
remove these instanced props. They're getting phased out. You can re-add them later when we have the subtype in.